PicPi/PictureFrameDetailsViewController.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  PictureFrameDetailsViewController.swift
+//  PicPi
+//
+//  Created by Mustafa T. Mohammed on 1/2/21.
+//
+
+import Foundation
+import Promises
+import Kingfisher
+
+class PictureFrameDetailsViewController: UICollectionViewController,UICollectionViewDelegateFlowLayout  {
+  
+  private let pictureFrame: PictureFrame
+  private var media = [Photo]()
+  
+  init(pictureFrame: PictureFrame, collectionViewLayout :UICollectionViewFlowLayout  = UICollectionViewFlowLayout()) {
+    self.pictureFrame = pictureFrame
+    
+    super.init(collectionViewLayout: collectionViewLayout)
+  }
+  
+  @available(*, unavailable)
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    
+    collectionView.register(PictureFrameDetailsThumbnailCollectionViewCell.self)
+    collectionView.register(PictureFrameDetailsPlusSignCollectionViewCell.self)
+    collectionView.backgroundColor = .white
+    title = pictureFrame.name
+    
+    /// asking the PictureFrame for its content
+    pictureFrame.content().then { (content) in
+      self.media = content.media
+      self.collectionView.reloadData()
+    }
+    
+  }
+  override func viewDidAppear(_ animated: Bool) {
+    super.viewDidAppear(true)
```

Comment:
nit: pass `animated` vs `true`

```suggestion
    super.viewDidAppear(animated)
```